tests/app/views/handlers/test_view_submitted_response_pdf.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta, timezone
+from unittest.mock import Mock
```

Comment:
Dont see `Mock` getting used in here ?

Reply:
Removed, we really need to get those pylint cards in, as lint should fail for these but don't atm 😞 .